.github/workflows/pr.yml

```diff
@@ -21,13 +21,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - shell: bash
         run: |
-          ./build/win-x64/test/MinSizeRel/nakama-test || {
+          ./out/test/win-x64/nakama-test.exe || {
```

Comment:
I think packaging scripts currently expect everything in "out" to be a release artifacts

Reply:
@redbaron yep this was actually a misconfigured path (fixed to `out/test`)